web/js/wv.main.js

```diff
@@ -62,19 +61,21 @@ import NotificationsUI from './notifications/notifications.ui';
 // UI
 import {loadingIndicator} from './ui/ui.indicator'; // not a class, export object
 // Link
-import LinkModel from './link/link.model';
-import LinkUI from './link/link.ui';
-import LinkInfo from './link/link.info';
+import linkModel from './link/link.model';
+import linkUi from './link/link.ui';
 // Projections
-import projectionParser from './proj/proj';
-import ProjectionModel from './proj/proj.model';
-import ProjectionUI from './proj/proj.ui';
-import ProjectionChange from './proj/proj.change';
+import projectionParser from './projection/projection';
```

Comment:
What do you think about combining each folder into a single object? That would match how the code is built right now where each file in the folder inherits and extends from the parent object. So rather than importing each file separately, we can import the whole module in one line;
```
import projection from './projection/projection';
```
And then references like this;
```
wv.projection.model
wv.projection.ui
wv.projection.change
```
Rather than giving them new names, we can drop the `wv.` from existing references and they become;
```
projection.model
projection.ui
projection.change
```

That's what I did with `util` and `ui`. Here's what I mean;

1) Named each module and submodule with the name of it's key on the original object. Parent: 
https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/util.js#L11
Submodule: https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/browser.js#L3

2) Imported the submodules into the parent module: https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/util.js#L7-L9

3) Exported the sub-modules on the parent object: https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/util.js#L15-L18

What do you think about doing that with projection, layers, and other modules as well? It might reduce the chance of crossing wires by changing so many references.